frontend/src/libs/components/input/input.tsx

```diff
@@ -18,23 +18,30 @@ type Properties<T extends FormFieldValues> = {
   type?: 'text' | 'email' | 'password';
   rowsCount?: number;
   maxLength?: number;
+  autoComplete?: 'off' | 'on';
+  required?: boolean;
```

Comment:
```suggestion
  isRequired?: boolean;
```
Rule B19 in quality criteria

Reply:
done